src/compiling/lexing/reader.rs

```diff
@@ -10,18 +10,18 @@ pub enum ReadMode {
 }
 
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
I'd just remove the serde's serialize and deserialize

Reply:
im not sure if there is a way to remove it without reverting #66

Reply:
im not sure if there is a way to do that without reverting #66